### ISSUE TITLE
Event detail link opening from mobile taking us directly to byte listing page

### ISSUE THREAD

**Issue by VayuzBidyut** [manojnegi07/social-external#1589]
Event link example: [https://events.vayuz.com/event-detail/demo-saturday-87-x-bridge-2.0/ZXZlbnRfMTcxNDcxMzEzOTU0Mg==](url)

**Comment by manojnegi07** [comment]
Yes because the domain of event is diffent then social so if user is not loggedin on event then it will take user to social first . This flow will be removed in sveltekit
